code/src/lib/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export const Button = ({ title,  background }) => {
+  return (
+    <SubmitButton>{title}</SubmitButton>
+  )
+}
```

Comment:
When I commented on your BarcodeButton.js, I didn't realize you had this button as well. If this is the button you want to use, you should be able to use it in BarcodeButton.js by importing it and mounting it:
import { Button } from "../lib/Button"

```
<Button
  type='button'
  title="Show scanner"
  onClick={() => setShowScanner(true)}
/>
```